app/scripts/modules/core/src/pipeline/config/stages/bakeManifest/helm/BakeHelmConfigForm.tsx

```diff
@@ -4,14 +4,25 @@ import { StageConfigField } from '../../common/stageConfigField/StageConfigField
 import { CheckboxInput, TextInput } from 'core/presentation';
 import { MapEditor } from 'core/forms';
 import { IArtifact, IExpectedArtifact, IPipeline } from 'core/domain';
-import { StageArtifactSelectorDelegate } from 'core/artifact';
+import { excludeAllTypesExcept, ArtifactTypePatterns, StageArtifactSelectorDelegate } from 'core/artifact';
 import { IFormikStageConfigInjectedProps } from '../../FormikStageConfig';
 
 interface IBakeHelmConfigFormProps {
   updatePipeline: (pipeline: IPipeline) => void;
 }
 
 export class BakeHelmConfigForm extends React.Component<IBakeHelmConfigFormProps & IFormikStageConfigInjectedProps> {
+  private static readonly excludedArtifactTypes = excludeAllTypesExcept(
+    ArtifactTypePatterns.BITBUCKET_FILE,
+    ArtifactTypePatterns.CUSTOM_OBJECT,
+    ArtifactTypePatterns.GCS_OBJECT,
+    ArtifactTypePatterns.GITHUB_FILE,
+    ArtifactTypePatterns.GITLAB_FILE,
+    ArtifactTypePatterns.S3_OBJECT,
+    ArtifactTypePatterns.HELM_CHART,
+    ArtifactTypePatterns.HTTP_FILE,
+  );
```

Comment:
@dbyron0 I'm going to need my resident Helm artifact expert to give this a once-over and make sure I didn't exclude any artifact types that you think folks could be using as input to the Bake (Manifest) stage for Helm. Complete list of artifact types is [here](https://github.com/spinnaker/deck/blob/master/app/scripts/modules/core/src/artifact/ArtifactTypes.ts#L5).

Thank you! 🙏

Reply:
Does EMBEDDED_BASE64 make sense?  Seems like it could.  I'm curious about JENKINS_FILE, IVY_FILE and MAVEN_FILE too, but know even less about those.  Could a helm chart tar ball be living behind them?

Reply:
Thanks for the quick reply, @dbyron0! I think you're right that `EMBEDDED_BASE64` could be fair game (just pushed a commit to include that type), but I think Ivy/Maven artifacts are for JAR files so I think we can safely exclude those for now.